src/blocks.rs

```diff
@@ -0,0 +1,188 @@
+    },
+    Block {
+        name: "brown_terracotta",
+        texture_names: [
+            "brown_terracotta",
+            "brown_terracotta",
+            "brown_terracotta",
+            "brown_terracotta",
+            "brown_terracotta",
+            "brown_terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "cyan_terracotta",
+        texture_names: [
+            "cyan_terracotta",
+            "cyan_terracotta",
+            "cyan_terracotta",
+            "cyan_terracotta",
+            "cyan_terracotta",
+            "cyan_terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "gray_terracotta",
+        texture_names: [
+            "gray_terracotta",
+            "gray_terracotta",
+            "gray_terracotta",
+            "gray_terracotta",
+            "gray_terracotta",
+            "gray_terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "light_gray_terracotta",
+        texture_names: [
+            "light_gray_terracotta",
+            "light_gray_terracotta",
+            "light_gray_terracotta",
+            "light_gray_terracotta",
+            "light_gray_terracotta",
+            "light_gray_terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "orange_terracotta",
+        texture_names: [
+            "orange_terracotta",
+            "orange_terracotta",
+            "orange_terracotta",
+            "orange_terracotta",
+            "orange_terracotta",
+            "orange_terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "red_terracotta",
+        texture_names: [
+            "red_terracotta",
+            "red_terracotta",
+            "red_terracotta",
+            "red_terracotta",
+            "red_terracotta",
+            "red_terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "terracotta",
+        texture_names: [
+            "terracotta",
+            "terracotta",
+            "terracotta",
+            "terracotta",
+            "terracotta",
+            "terracotta",
+        ],
+        is_solid: true,
+    },
+    Block {
+        name: "yellow_terracotta",
+        texture_names: [
+            "yellow_terracotta",
+            "yellow_terracotta",
+            "yellow_terracotta",
+            "yellow_terracotta",
+            "yellow_terracotta",
+            "yellow_terracotta",
+        ],
+        is_solid: true,
+    },
+];
+
+pub static AIR: u8 = 0;
+pub static GRASS: u8 = 1;
+pub static DIRT: u8 = 2;
+pub static STONE: u8 = 3;
+pub static BEDROCK: u8 = 4;
+pub static RED_SAND: u8 = 5;
+pub static BROWN_TERRACOTTA: u8 = 6;
+pub static CYAN_TERRACOTTA: u8 = 7;
+pub static GRAY_TERRACOTTA: u8 = 8;
+pub static LIGHT_GRAY_TERRACOTTA: u8 = 9;
+pub static ORANGE_TERRACOTTA: u8 = 10;
+pub static RED_TERRACOTTA: u8 = 11;
+pub static TERRACOTTA: u8 = 12;
+pub static YELLOW_TERRACOTTA: u8 = 13;
```

Comment:
Is there a more DRY way of creating these constants?

Reply:
Use an enum:
```Rust
#[repr(u8)]
enum MyEnum {
    AIR = 0,
    GRASS,
    DIRT,
    ...
}
```

Reply:
True. Much better than the other approach.